### PR TITLE
Fix: Address lookup validation

### PR DESCRIPTION
**Pull request by Adnan-cds**
## What does this change?
The LocalGov address lookup element's validation process is losing existing form error messages set by other parts of a form.  Some of its own error messages are also getting lost.  This change fixes this.

I have also updated the wording for error messages shown on required fields like Address line 1 and Postcode.  Instead of saying "Postcode field is required", it now says "You must enter the postcode."  Error message wording courtesy of @DavidBAsch 

## How to test
- Create a new Webform.
- Add a set of radio buttons.  Mark it as `required`.
- Add a LocalGov address lookup element.
- Set the visibility condition of the address lookup element dependent on one of the radio buttons.
- Submit the Webform without selecting any of the radio buttons.  At the moment, form submission completes and an error message about the radio button is displayed in the **confirmation** page rather than on the form page where the form elements are.  Once this fix is applied, the form submission fails as expected.

### CODE REVIEW

**Comment by Adnan-cds** [comment]
Marking this pull request as "Draft" to add further improvements.

File: src/Element/UKAddressLookup.php [hunk 1]
```diff
@@ -59,6 +59,9 @@ public static function getCompositeElements(array $element) {
     $element_list['address_1']['#prefix'] = '<div class="js-address-entry-container">';
     $element_list['postcode']['#suffix'] = '</div>';
 
+    // Custom error message for address line 1.
+    $element_list['address_1']['#required_error'] = t('You must enter an address.');
```
Comment (ekes) [inline src/Element/UKAddressLookup.php:63]: Not blocking this. But might be worth moving to StringTranslationTrait at some point.

**Review by ekes — APPROVED** [review]
Woo that was niche, even to test.

But does as described, and automated tests look good.

File: src/Element/UKAddressLookup.php (hunk 1 above)
Comment (Adnan-cds) [reply src/Element/UKAddressLookup.php:63]: Yeah.  It caused me some unease as well.  But eventually stuck to the t() function for consistency with the rest of the file.  I have just raised a [ticket](https://github.com/localgovdrupal/localgov_forms/issues/99) to replace the entire lot.

**Comment by Adnan-cds** [comment]
> Woo that was niche, even to test.

Yeah, spotted by @DavidBAsch :D  The Address lookup element is a lot better because of him.

Thanks for the approval.  Merging...

**Pull request merged**